docs/terminology.md

```diff
@@ -0,0 +1,48 @@
+# XLA Terminology
+
+There are several terms that are used in the context of XLA, MLIR, LLVM, and other related technologies. Below is a partial list of these terms and their definitions.
+
+- **OpenXLA**
```

Comment:
Can we order these:

- OpenXLA
- XLA
- PJRT
- StableHLO
- CHLO
- VHLO
- MHLO
- HLO
- MLIR
- LLVM


Also recommendations for CHLO:

- CHLO is a collection of higher level operations which are optionally decomposable to StableHLO.